docs/addons/jwt.md

```diff
@@ -190,7 +190,7 @@ class LoginController extends BaseController
         }
 
         // Get the credentials for login
-        $credentials             = $this->request->getJsonVar(setting('Auth.validFields'));
+        $credentials             = $this->request->getJsonVar(shieldSetting('Auth.validFields'));
```

Comment:
I believe this should be in snake case per our coding standards: `shield_setting`